### PR TITLE
Update accessibility.mdx

### PR DESCRIPTION
**Pull request by mbgower**
Update to Accessibility content.
Still need to upload images

### CODE REVIEW

**Review by tay1orjones — APPROVED** [review]

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 1]
```diff
@@ -19,93 +19,105 @@ import {
...
-
-### Automated test
+## What Carbon provides
+
+The Carbon header bakes in keyboard interaction, including a skip-to-main-content
+mechanism. Carbon also supports assistive technologies such as screen readers by setting labeling and page structure.
+
+### Keyboard interaction
+
+Each element in the header can be reached by `Tab` key. A "Skip to main content" link appears when a
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:42]:
```suggestion
Each element in the header can be reached by the `Tab` key. A "Skip to main content" link appears when a
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 2]
```diff
@@ -19,93 +19,105 @@ import {
...
+
+### Regions
+
+<Row>
+<Column colLg={8}>
+
+![example of header keyboard interaction](images/header-accessibility-2.png)
+
+<Caption>
+  Carbon applies a header region around the whole header UI shell.
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:66]:
```suggestion
  Carbon applies a header region around the whole UI shell header.
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 3]
```diff
@@ -19,93 +19,105 @@ import {
...
+</Caption>
+
+</Column>
+</Row>
+
+### Labeling
+
+Carbon provides default names for each icon-only button in the header, and these names are revealed on hover or focus. Interactions for some header components are covered under other component topics: [search](/components/search/usage), [notifications](/components/notifications/usage), [switcher](components/UI-shell-right-panel/usage/#switcherswitcher).
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:74]:
```suggestion
Carbon provides default names for each icon-only button in the header, and these names are revealed on hover or focus. Interactions for some header components are covered under other component topics such as [search](/components/search/usage), [notifications](/components/notifications/usage), and the [UI shell right panel switcher](components/UI-shell-right-panel/usage/#switcher).  
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 4]
```diff
@@ -19,93 +19,105 @@ import {
...
+### Labeling
+
+Carbon provides default names for each icon-only button in the header, and these names are revealed on hover or focus. Interactions for some header components are covered under other component topics: [search](/components/search/usage), [notifications](/components/notifications/usage), [switcher](components/UI-shell-right-panel/usage/#switcherswitcher).  
+<Row>
+<Column colLg={8}>
+
+![annotation showing search, help, notification and app switcher icons](images/header-accessibility-3.png)
+
+<Caption>
+  Carbon provides default names for its icon-only components, and reveals them on hover or focus.
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:81]:
```suggestion
  Carbon provides default names for its icon-only components and reveals them on hover or focus.
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 5]
```diff
@@ -19,93 +19,105 @@ import {
...
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![Annotated header region stating 'Header functionality, roles and names match Carbon defaults'](images/header-accessibility-4.png)
+
+<Caption>Annotate your header, even if your design matches the default Carbon UI shell header behavior.</Caption>
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:97]:
```suggestion
<Caption>Annotate the header, even if your design matches the default Carbon UI shell header behavior.</Caption>
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 6]
```diff
@@ -19,93 +19,105 @@ import {
...
+Once a product-specific header annotation exists, individual product pages only need to annotate the header if something differs.
+
+Where the header deviates from the default Carbon behavior or labeling, it should be annotated. Such annotations could call out different labels for icons or indicate interaction changes such as keyboard navigation.
+
+<Row>
+<Column colLg={8}>
+
+![an annotation stating the search bar is persistently expanded](images/header-accessibility-5.png)
+
+<Caption>Annotate if your design modifes the default keyboard or component interaction.</Caption>
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:111]:
```suggestion
<Caption>Annotate if the design modifies the default keyboard or component interaction.</Caption>
```

File: src/pages/components/UI-shell-header/accessibility.mdx [hunk 7]
```diff
@@ -19,93 +19,105 @@ import {
...
+</Row>
+
+
+## Development considerations
+
+Keep these considerations in mind if you are modifying Carbon or creating a
+custom component.
+
+- Carbon uses the HTML 5 `header` component instead of an ARIA landmark to set the region.
+- Carbon includes the Skip to main content link as the first item on the page with a `tabindex="0"`, but hides it through CSS until it receives focus.
```
Comment (laurenmrice) [inline src/pages/components/UI-shell-header/accessibility.mdx:123]:
```suggestion
- Carbon includes the "Skip to main content" link as the first item on the page with a `tabindex="0"`, but hides it through CSS until it receives focus.
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
@mbgower Looks good! Just a couple of typos and content comments.

**Comment by mbgower** [comment]
@laurenmrice I accepted all those changes.

**Review by laurenmrice — APPROVED** [review]
Looks good to me!

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**